README.md

```diff
@@ -83,10 +77,7 @@ $profile = $personaClient->getUserByGupid("google:123", "some token");
 // create an instance of the client
 $personaClient = new Talis\Persona\Client\Login(array(
     'persona_host' => 'http://persona',
-    'persona_oauth_route' => '/oauth/tokens',
-    'tokencache_redis_host' => 'localhost',
-    'tokencache_redis_port' => 6379,
-    'tokencache_redis_db' => 2
+    'persona_oauth_route' => '/oauth/tokens'
```

Comment:
The user would need to pass in the caching object & the construct would take the configuration.